packages/header-checker-lint/test/header-checker-lint.ts

```diff
@@ -15,7 +15,6 @@
 import {resolve} from 'path';
 // eslint-disable-next-line node/no-extraneous-import
 import {Probot, createProbot, ProbotOctokit} from 'probot';
-import {Octokit} from '@octokit/rest';
 import snapshot from 'snap-shot-it';
```

Comment:
just confirming, it seems like we weren't using this?

Reply:
I used Octokit in the previous code, but it started to fail with sinon.assert. I changed it to ProbotOctokit and now it's passing.